templates/communitypage.html

```diff
@@ -0,0 +1,161 @@
+      </div>
+    </div>
+  </div>
+  <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+  <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+</div>
+  <div class="row my-3" style="padding-bottom:30px;">
+    <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+  <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+  <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+<div class="container my-auto" style="text-align: center; padding-bottom: 30px;">
+  <hr>
+  <h4 id="subheading" style="text-align: center;">Subscribe to our newsletter to get latest blog post links</h4>
+  <div class="box" style="position: relative; margin-left: auto !important; margin-right: auto !important; padding-top: 20px;">
+      <input style = "height:40px; width:350px;" type="search" placeholder="Enter your mail id" aria-label="Search Communtiies">
+      <button class="btn btn-outline-warning my-2 my-sm-0" type="submit">Mail me</button>
+    </div>
+</div>
+<!-- Footer starts here -->
```

Comment:
Hey @AdityaAshvin as you have created a new file for footer can we please remove these codes and simply render `footer.html` here.